src/Symfony/Component/ClassLoader/ClassFinder.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ClassLoader;
+
+/**
+ * ClassFinder.
+ *
+ * @author Gyula Sallai <salla016@gmail.com>
+ */
+class ClassFinder
```

Comment:
Adding a new class here looks wrong to me, as the whole component is deprecated

Reply:
Yes, component is deprecated and will be removed in 4.0 just like the functionality of Kernel lookup inside KERNEL_DIR, where this code is used. 
I supposed that it probably worth to reuse the code (with introducing new class but without introducing any kind of functionality) instead of copy-pasting since this 2 code blocks have same roadmap.